data/sql/updates/pending_db_world/0000_00_00_00_lolz.sql

```diff
@@ -0,0 +1,144 @@
+(25222, 1, 1, 'I thought we had enough in your line of work, but it looks like we have some openings.  Report to the civilian liaison.', 12, 7, 100, 273, 0, 0, 24360, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 1, 2, 'I do have some openings in your line of work.  Report to the civilian liaison.', 12, 7, 100, 273, 0, 0, 24359, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 1, 3, 'Yeah.  We could use some more of you.  Report to the civilian liaison for work assignment.', 12, 7, 100, 273, 0, 0, 24358, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 2, 0, 'Not anymore!  Here''s your sword.  Report to the barracks for duty!', 12, 7, 100, 397, 0, 0, 24363, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 2, 1, 'Afraid not, friend.  Here''s your sword and shield.  Report to the barracks for duty.', 12, 7, 100, 397, 0, 0, 24365, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 2, 2, 'You''re breaking my heart - I''ve quotas to fill, you know?  Can you wield a sword?  Off to the barracks.', 12, 7, 100, 397, 0, 0, 24367, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 2, 3, 'With a sword arm like that?  I''m afraid we need you in the front lines, my friend.  Report to the barracks.', 12, 7, 100, 397, 0, 0, 24366, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 2, 4, 'Looks like we have room for one more... no, my mistake.  That''s a seven not a one.  Hope you''re good with a sword - report to the barracks.', 12, 7, 100, 397, 0, 0, 24368, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 3, 0, 'Next, please!', 12, 7, 100, 22, 0, 0, 24370, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 3, 1, 'Keep it moving, people.', 12, 7, 100, 22, 0, 0, 24369, 'Recruitment Officer Carven to Generic Quest Trigger - LAB'),
+(25222, 3, 2, 'Next!', 12, 7, 100, 22, 0, 0, 24357, 'Recruitment Officer Carven to Generic Quest Trigger - LAB');
+
+DELETE FROM `waypoints` WHERE `entry`=25220;
+INSERT INTO `waypoints` (`entry`, `pointid`, `position_x`, `position_y`, `position_z`, `point_comment`) VALUES 
+(25220, 1, 2253.644, 5195.469, 11.40062, 'Civilian Recruit'),
+(25220, 2, 2254.098, 5196.36, 11.40062, 'Civilian Recruit'),
+(25220, 3, 2277.823, 5238.724, 11.45096, 'Civilian Recruit'),
+(25220, 4, 2279.217, 5241.407, 11.45096, 'Civilian Recruit'),
+(25220, 5, 2280.84, 5244.218, 11.45719, 'Civilian Recruit'),
+(25220, 6, 2282.603, 5245.736, 11.36353, 'Civilian Recruit'),
+(25220, 7, 2284.867, 5246.299, 11.45096, 'Civilian Recruit'),
+(25220, 8, 2287.469, 5245.929, 11.45096, 'Civilian Recruit'),
+(25220, 9, 2289.469, 5244.898, 11.45096, 'Civilian Recruit'),
+(25220, 10, 2291.772, 5243.933, 11.45096, 'Civilian Recruit'),
+(25220, 11, 2294.129, 5242.708, 11.45096, 'Civilian Recruit'),
+-- (25220, 12, 2298.062, 5241.932, 12.3176, 'Civilian Recruit'),
```

Comment:
?

Reply:
Left for reference, the commented out point12 is blizzlike (they end up standing on the table), the custom point 12 is a bit back, so they don't do that. I have left a comment.